packages/hooks/src/useViewportSize.ts

```diff
@@ -13,14 +13,12 @@ function getViewportSize(): ViewportSize {
   };
 }
 
-export default function useViewportSize(): ViewportSize | null {
-  const [viewportSize, setViewportUpdateVal] = useState<ViewportSize | null>(
-    null,
+export default function useViewportSize(): ViewportSize {
+  const [viewportSize, setViewportUpdateVal] = useState<ViewportSize>(
+    getViewportSize(),
```

Comment:
is `window` always defined on initial render? IIRC we ran into issues with this, but they may not be a concern anymore. 
Could you add tests to verify this?

Reply:
`window` was defined on initial render when manually testing. Will add a test to verify it shortly